modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/Sidebar.scss

```diff
@@ -5,7 +5,7 @@
 		box-shadow: none;
 
 		.sidebar-body {
-			height: calc(100vh - 224px);
+			height: calc(100vh - 273px);
```

Comment:
Hi @EliveltonR18, The difference between 273 and 224 is 13px, but I think you can use our baseline of 16px here

Reply:
Hi @interaminense, I did the change, but the sidebar yet be cutted.
![Screenshot from 2021-05-03 11-46-45](https://user-images.githubusercontent.com/38770741/116899042-7a386280-ac0d-11eb-8aaa-f25bd24b4870.png)

Reply:
@interaminense I changed the height to 272px, I think it's ok now.